server/container/cli/src/main/java/org/apache/james/cli/type/CmdType.java

```diff
@@ -22,44 +22,44 @@
  * Enumeration of valid command types.
  */
 public enum CmdType {
-    ADDUSER("adduser", "username","password"),
-    REMOVEUSER("removeuser", "username"),
-    LISTUSERS("listusers"),
-    ADDDOMAIN("adddomain", "domainname"),
-    REMOVEDOMAIN("removedomain", "domainname"),
-    CONTAINSDOMAIN("containsdomain", "domainname"),
-    LISTDOMAINS("listdomains"),
-    LISTMAPPINGS("listmappings"),
-    LISTUSERDOMAINMAPPINGS("listuserdomainmappings", "user","domain"),
-    ADDADDRESSMAPPING("addaddressmapping", "user","domain", "fromaddress"),
-    REMOVEADDRESSMAPPING("removeaddressmapping", "user","domain", "fromaddress"),
-    ADDREGEXMAPPING("addregexmapping", "user","domain", "regex"),
-    REMOVEREGEXMAPPING("removeregexmapping", "user","domain", "regex"),
-    SETPASSWORD("setpassword", "username","password"),
-    COPYMAILBOX("copymailbox", "srcbean","dstbean"),
-    DELETEUSERMAILBOXES("deleteusermailboxes", "user"),
-    CREATEMAILBOX("createmailbox", "namespace", "user", "name"),
-    LISTUSERMAILBOXES("listusermailboxes", "user"),
-    DELETEMAILBOX("deletemailbox", "namespace", "user", "name"),
-    GETSTORAGEQUOTA("getstoragequota", "quotaroot"),
-    GETMESSAGECOUNTQUOTA("getmessagecountquota", "quotaroot"),
-    GETQUOTAROOT("getquotaroot", "namespace", "user", "name"),
-    GETMAXSTORAGEQUOTA("getmaxstoragequota", "quotaroot"),
-    GETMAXMESSAGECOUNTQUOTA("getmaxmessagecountquota", "quotaroot"),
-    SETMAXSTORAGEQUOTA("setmaxstoragequota", "quotaroot", "max_message_count"),
-    SETMAXMESSAGECOUNTQUOTA("setmaxmessagecountquota", "quotaroot", "max_storage"),
-    SETDEFAULTMAXSTORAGEQUOTA("setdefaultmaxstoragequota", "max_storage"),
-    SETDEFAULTMAXMESSAGECOUNTQUOTA("setdefaultmaxmessagecountquota", "max_message_count"),
-    GETDEFAULTMAXSTORAGEQUOTA("getdefaultmaxstoragequota"),
-    GETDEFAULTMAXMESSAGECOUNTQUOTA("getdefaultmaxmessagecountquota"),
-    REINDEXMAILBOX("reindexmailbox", "namespace", "user", "name"),
-    REINDEXALL("reindexall"),
-    GETSIEVEQUOTA("getsievequota"),
-    SETSIEVEQUOTA("setsievequota", "quota"),
-    REMOVESIEVEQUOTA("removesievequota"),
-    GETSIEVEUSERQUOTA("getsieveuserquota", "username"),
-    SETSIEVEUSERQUOTA("setsieveuserquota", "username", "quota"),
-    REMOVESIEVEUSERQUOTA("removesieveuserquota", "username");
+    ADDUSER("AddUser", "username","password"),
+    REMOVEUSER("RemoveUser", "username"),
+    LISTUSERS("ListUsers"),
+    ADDDOMAIN("AddDomain", "domainName"),
+    REMOVEDOMAIN("RemoveDomain", "domainName"),
+    CONTAINSDOMAIN("ContainsDomain", "domainName"),

```

Comment:
camelCase for arguments seems inconsistent with some actual snake_case use (see for example max_message_count, max_storage). Can you make it more uniform?